CHANGELOG.md

```diff
@@ -19,3 +26,7 @@
 - Added cloud-init support.
 - Added support for Windows targets.
   - Can inject powershell script to set a random password and enable WinRM
+
+
+steele.justin@gmail.com
+3s!T8mRb6xbz%y
```

Comment:
What is this tag for in the CHANGELOG?

Reply:
not sure what that is, but good catch.  I'll push another commit without that in there.